HOWTO.md

```diff
@@ -591,3 +635,59 @@ Please mind how the configuration file must be specifically named (e.g. `-` VS
     ```
 
 3. Run the client: `fdo-client-linuxappp`
+
+### Environmental Variables
+
+Here is a list of environmental variables, their meaning and suggested values. 
+Note: Contributors/Developers are encouraged to amend to this list of env variables 
+as they come across or write new variables.
+
+1. `LOG_LEVEL` 
+- Usage: This variable is used to provide the log level while running any of the 
+servers or client binary or systemd service.
+- Suggested values:
+`trace` , `info`, `debug`, `error`
+
+2. `MANUFACTURING_SERVER_CONF`
+- Usage: This variable is used to provide path to `manufacturing_server.yml` config file
+- Suggested values: `/path/to/manufacturing_server.yml`
+
+
+3. `DI_MFG_STRING_TYPE` 
+- Usage: This variable is used to provide what type of device info is needed from the device.
+This can be used for device identification.
+As of today this variable has two values mentioned below.
+- Suggested values: `SerialNumber` or `StructuredDeviceInfo`
+
+Note: 
```

Comment:
Align this note  and the following explanation to the right so that this section appears under `3. DI_MFG_STRING_TYPE`